components/inputs/input.css

```diff
@@ -0,0 +1,45 @@
+/* Styling for input components group */
+.input-group {
+  margin: var(--scale-sm);
+  padding: var(--scale-sm);
+}
+
+/* Common Styling for input boxes */
+.input-group input[type="text"],
+input[type="password"],
+input[type="email"],
+input[type="tel"],
+input[type="search"],
+input[type="url"] {
```

Comment:
was attribute part in selector required?  try checking code without it and change it if it works without attribute part

Reply:
For inputs like checkboxes and radio btns it is breaking. Hence I added this for all text like inputs.